.circleci/config.yml

```diff
@@ -39,3 +38,48 @@ jobs:
             make everything
             make run_tests
             ./teeworlds_srv shutdown
+
+  publish-github-release:
+    docker:
+      - image: circleci/golang
+    steps:
+      - checkout
+      - run:
+          name: Prepare for deploy
+          command: |
+            sudo apt-get update -y
+            sudo apt-get install cmake libfreetype6-dev libsdl2-dev -y
+            git submodule update --init
+            go get -u github.com/tcnksm/ghr
+      - run:
+          name: Build bam
+          command: |
+            git clone https://github.com/matricks/bam.git ~/bam
+            cd ~/bam/
+            git reset --hard f012dd9a3e38295b8a45af5a101d29573381f169
```

Comment:
I think it is wisest to pull https://github.com/teeworlds/bam

Reply:
I copied that code from you, take a look at the building with bam part!

Reply:
I did not write that. It doesn't matter too much anyway :)